data/json/npcs/refugee_center/surface_staff/NPC_free_merchant_teamster.json

```
@@ -109,9 +107,7 @@
         "text": "I think I've got the evidence you were looking for.",
         "condition": {
           "and": [
-            {
-              "not": { "compare_num": [ { "global_val": "var", "var_name": "godco_freemerch_representative_dead" }, "==", { "const": 0 } ] }
-            },
+            { "not": { "math": [ "godco_freemerch_representative_dead", "==", "0" ] } },
```

```suggestion
            { "math": [ "godco_freemerch_representative_dead", "!=", "0" ] },
```
There's another one like this below